src/main/scala/io/smartdatalake/util/hive/HiveUtil.scala

```diff
@@ -216,6 +216,7 @@ private[smartdatalake] object HiveUtil extends SmartDataLakeLogger {
                     hiveDb: String, partitions: Seq[String], saveMode: SaveMode,
                     hdfsOutputType: OutputType = OutputType.Parquet, numInitialHdfsPartitions: Int = -1): Unit = {
     implicit val sss = session
+    require(!outputDir.equals(""), s"Can not write HiveTable ${hiveDb}.${hiveTable} because path was not set.")
```

Comment:
Can it really happen that we pass an empty string? For me it's unusal to test for empty string, as that should normally be an Option[String] with value None...

Reply:
You're right, removed the check completely. It didn't make much sense, the string can not be empty.